### ISSUE TITLE
browser/filter by address book

### ISSUE THREAD

**Issue by quantenschaum** [nextcloud/contacts#2132]
**Is your feature request related to a problem? Please describe.**
I want to browse contacts, that are in *one* address book, but NC is always displaying *all* contacts from all address books.

**Describe the solution you'd like**
I would like to select/enable address books, such that only contacts in the selected/enabled address books are show in the NC web UI. It should behave like the calendars do, where I can show/hide each individual calendar.

**Describe alternatives you've considered**
To achieve this, I currently use CardBook in Thunderbird.

**Comment by kosssi** [comment]
:+1: 

I manage more than 700 contacts on Nextcloud with about ten address books. The fact of filtering by address book will facilitate my research and the loading time of the application.

**Comment by tobefrank1** [comment]
I support this!

Like groups, it would be very useful to display address books separately. 
We have several address books with different functions customer, supplyer, staff, private ... .
The only way to work with one is do disable the others.  That is not very useful.  

But if one could use them like the groups are handelt, that would be perfect.

**Comment by hamza221** [comment]
Hello, Thank you for your suggestion, we accept this feature and even though it's on our roadmap for the moment we think that it would make a good improvement to the contacts app.

**Comment by Cisco30** [comment]
+ 1

**Comment by thomas292** [comment]
this will be a great and needed addition.

**Comment by derWahnsinn** [comment]
This would be much needed.
Sharing for address books is already possible, but actually identifying which contacts are being shared is giving me headaches.

**Comment by Malachiel87** [comment]
+1 need this for view private contacts and business contacts book

**Comment by worldofjoni** [comment]
Duplicate of #1703?

**Comment by hamza221** [comment]
> Duplicate of [#1703](https://github.com/nextcloud/contacts/issues/1703)?


yes indeed, let's track it there. I'll close this issue as duplicate since it's newer
